packages/js/product-editor/src/blocks/generic/select/edit.tsx

```diff
@@ -24,7 +24,6 @@ export function Edit( {
 		property,
 		label,
 		note,
-		placeholder,
```

Comment:
We should also remove it from the `block.json` attributes list.

Reply:
Done with 7cc4e11d397d28e6a9a06d1229182286bade10e3